### ISSUE TITLE
Don't use flatmap-stream (installing appcenter fails)

### ISSUE THREAD

**Issue by Minishlink** [Microsoft/appcenter-cli#508]
Hello,

The package `flatmap-stream` has been unpublished by npm because of potential malicious code. See https://github.com/dominictarr/event-stream/issues/116

Thus, `npm install -g appcenter-cli` fails right now...

**Comment by owenniblock** [comment]
Hi @Minishlink - thanks for reporting this. We're looking in to a solution for this.

**Comment by MrTravisB** [comment]
Should be able to just lock to `event-stream@3.3.4` instead of `^3.3.4`?

**Comment by tedrog36** [comment]
I am having the same issue I believe trying to install appcenter-cli

```
389 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "appcenter-cli"
390 verbose node v10.13.0
391 verbose npm  v6.4.1
392 error code ENOVERSIONS
393 error No valid versions available for flatmap-stream
```

**Comment by amchew** [comment]
Hi @Minishlink, @tedrog36 and @MrTravisB, thank you for notifying us! We're investigating this issue now.

**Comment by tedrog36** [comment]
Thanks @amchew!  Seems to be working now.

**Comment by owenniblock** [comment]
Thanks everyone! We've released a version fixed at `3.3.4` just in case [release notes are here](https://github.com/Microsoft/appcenter-cli/releases/tag/v1.1.8)

I've also written up an internal memo about this vulnerability so we can dissect what happened and see if there is anything better we can do to identify and mitigate against things like this happening in future.

@amchew - closing this 🙂

**Comment by amchew** [comment]
Awesome to hear @tedrog36, and thanks for the quick fix @owenniblock! FYI @Minishlink and @MrTravisB, we've just re-released v1.1.8, you should be able to install appcenter-cli now. Please test this out, and let us know if you face any issues.

Thank you!
